File: Java/OPSJLib/src/ops/ReliabilityQoS.java
/*
 * ReliabilityQoS.java
 *
 * Created on den 21 september 2007, 11:24
 *
 * To change this template, choose Tools | Template Manager
 * and open the template in the editor.
 */

package ops;

/**
 *
 * @author xangr
 */
public class ReliabilityQoS extends QualityOfService
{
    private int replyPort;
    private int nrOfResends = 1;
    private long resendInterval = 100;
    /** Creates a new instance of ReliabilityQoS */
    public ReliabilityQoS()
    {
        
    }
    public String getQoSID()
    {
        return "reliability";
    }
    
}

File: C#/Ops/ReliabilityQoS.cs
///////////////////////////////////////////////////////////
//  ReliabilityQoS.cs
//  Implementation of the Class ReliabilityQoS
//  Created on:      12-nov-2011 09:25:35
//  Author:
///////////////////////////////////////////////////////////

namespace Ops 
{
	public class ReliabilityQoS : QualityOfService
    {
#pragma warning disable 414,169  // variables not used
        private int nrOfResends = 1;
        private int replyPort;
        private long resendInterval = 100;
#pragma warning restore 414,169

        public ReliabilityQoS()
        {
		}

        public string GetQoSID()
        {
            return "reliability";
        }

	}

}